packages/frontend/src/components/Game/GameBoard/Tile/index.test.jsx

```diff
@@ -2,31 +2,50 @@ import { setupTests } from 'helpers/tests';
 import Tile from '.';
 import { mockPhotos } from '__mocks__/api/mockPhotos';
 import { createTilesFromPhotos } from 'helpers';
+import userEvent from '@testing-library/user-event';
 
 describe('Tile component', () => {
   it('renders', () => {
-    const { screen } = setupTests(Tile);
+    const tile = createTilesFromPhotos(mockPhotos)[0];
+
+    const { screen } = setupTests(Tile, { props: { tile: tile } });
```

Comment:
For all the Tile tests, there could be a variable `const props = { tile };` or, if rendering a tile with props other than the defaults `const props = { tile: { ...tile, faceUp: true } };`, and then the test can be set up with `setupTests(Tile, { props });`